metrics/storage/blogbench_dockerfile/Dockerfile.in

```diff
@@ -5,7 +5,7 @@
 # Set up an Ubuntu image with 'blogbench' installed
 
 # Usage: FROM [image name]
-FROM @UBUNTU_REGISTRY@/ubuntu
+FROM @UBUNTU_REGISTRY@/ubuntu:20.04
```

Comment:
why this particular version?:

Reply:
Currently `ubuntu:latest` points to `ubuntu:20.04`. Should we stick with `latest`?

Reply:
According to https://hub.docker.com/_/ubuntu?tab=description, `latest` == "latest LTS", which seems like a reasonable value here (shame the label isn't `latest-lts` though ;)

Reply:
So for all Ubuntu images which I touched I can be using `latest` + ignore the rule  https://github.com/hadolint/hadolint/wiki/DL3007 ?

Reply:
Here and in all Dockerfile under metrics I changed to use `latest`.